cmake/cpu_features.cmake

```diff
@@ -30,7 +30,13 @@ if (ARCH_NATIVE)
     set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=native")
 
 elseif (ARCH_AARCH64)
-    set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=armv8-a+crc+simd+crypto+dotprod+ssbs")
+    CHECK_C_COMPILER_FLAG("-march=armv8-a+lse" HAS_ARMV8_LSE)
```

Comment:
`CHECK_C_COMPILER_FLAG` doesn't compile anything, it merely checks if the C compiler accepts the passed arguments. A ClickHouse build requires minimum versions of Clang and GCC, see "cmake/tools.cmake" for that, and these versions are rather new. We can simply assume that the compiler knows the `lse` flag.

Reply:
Yes, that make sense. Let me remove the checking and just add the "lse" into the feature list.